src/runtime/runtime.ts

```diff
@@ -336,8 +298,28 @@ export class TestnetRuntime extends Runtime {
     }
   }
 
+  static get provider(): nearAPI.providers.JsonRpcProvider {
+    return new nearAPI.providers.JsonRpcProvider(this.defaultConfig.rpcAddr);
+  }
+
+  /** 
+   * Get most recent Wasm Binary of given account.
+   * */ 
+  static async viewCode(account_id: string): Promise<Buffer> {
```

Comment:
Is there any reason for this to live in `TestnetRuntime` instead of `Runtime`?

Reply:
Because of the static issue I mentioned.  There isn't a notion of dynamic dispatch.  For example `this.provider` doesn't exist as a static on Runtime and if it did it wouldn't let testnet's method override it.

Reply:
This will be addressed in my work on caching which requires fixing this.